tests/disaster-recovery/regional-dr/test_failover_and_relocate.py

```diff
@@ -121,6 +131,39 @@ def test_failover_and_relocate(
         logger.info(f"Waiting for {wait_time} minutes to run IOs")
         sleep(wait_time * 60)
 
+        # Set cluster_kubeconfig value for 'drpc_obj' object to fetch the details without switching the cluster context
+        acm_cluster_kubeconfig = os.path.join(
+            config.clusters[get_active_acm_index()].ENV_DATA["cluster_path"],
+            config.clusters[get_active_acm_index()].RUN.get("kubeconfig_location"),
+        )
+        drpc_obj.cluster_kubeconfig = acm_cluster_kubeconfig
+
+        # Get lastGroupSyncTime before failover
+        drpc_data = drpc_obj.get()
+        last_group_sync_time = drpc_data.get("status").get("lastGroupSyncTime")
+        logger.info(
+            f"The value of lastGroupSyncTime before failover is {last_group_sync_time}."
+        )
+
+        # Verify lastGroupSyncTime before failover
+        time_format = "%Y-%m-%dT%H:%M:%SZ"
```

Comment:
Can we have a dedicated helper function/conftest for this check as it will be used/repeated in all failover/relocate tests be it UI or CLI?

Reply:
Hi
Do you think we can take it as an enhancement of this PR ? Not only the verification but also Identifying the correct drpc cr, getting the lastGroupSyncTime and verifying it can be done seperately in a function as a seperate PR.

Reply:
yes we can

Reply:
pls open a github issue to track it

Reply:
Done- https://github.com/red-hat-storage/ocs-ci/issues/8724